alembic.ini

```diff
@@ -29,7 +29,7 @@ script_location = db_migrate
 # are written from script.py.mako
 # output_encoding = utf-8
 
-sqlalchemy.url = postgres://codechecker@localhost:8764/codechecker
+sqlalchemy.url = postgres://codechecker@localhost:5432/codechecker
```

Comment:
The default Postgresql 5432 port is the new default port for CodeChecker db port.